gf-webapp-admin/src/component/rooms/roomSimplified/overlay/Overlay.tsx

```diff
@@ -17,7 +17,7 @@ export const Overlay = (props: OverlayProps) => {
         className={styles.info}
       >
         <h3>
-          {`${props.room.apartmentsType} room`}
+          {`${props.room.type} room`}
```

Comment:
U should separate UI and data layers. What about the special technical type "ROOM_NOT_READY" for example? Or What if we start write all enums with UPPERCASE, or it will consist TWO_WORDS?

Reply:
Done